frontend/kingdoms/kingdoms.css

```
@@ -0,0 +1,456 @@
+  --gold: #d4a64a;
+  --gold-deep: #a77e2e;
+  --emerald: #3d7a5f;
+  --bg: #1a1410;
+  --bg-2: #221812;
+  --surface: rgba(255, 240, 210, 0.06);
+  --surface-2: rgba(28, 18, 10, 0.8);
+  --line: rgba(212, 166, 74, 0.2);
+  --text: #f1e7cf;
+  --muted: #a89878;
+  --shadow: 0 22px 55px rgba(0, 0, 0, 0.55);
+  --font-display: "Cormorant Garamond", "Noto Sans Devanagari", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)